docs/content/policy-reference.md

```diff
@@ -175,9 +175,16 @@ The following table shows examples of how ``glob.match`` works:
 
 OPA provides two builtins that implement JSON Web Signature [RFC7515](https://tools.ietf.org/html/rfc7515) functionality.
 
-``io.jwt.encode_sign_raw()`` takes three JSON Objects (strings) as parameters and returns their JWS Compact Serialization. This builtin should be used by those that want maximum control over the signing and serialization procedure. It is important to remember that StringOrURI values are compared as case-sensitive strings with no transformations or canonicalizations applied. Therefore, line breaks and whitespaces are significant.
+``io.jwt.encode_sign_raw()`` takes three JSON Objects (strings) as parameters and returns their JWS Compact Serialization.
+ This builtin should be used by those that want maximum control over the signing and serialization procedure. It is
+ important to remember that StringOrURI values are compared as case-sensitive strings with no transformations or
+ canonicalizations applied. Therefore, line breaks and whitespaces are significant.
 
-``io.jwt.encode_sign()`` takes three Rego Objects as parameters and returns their JWS Compact Serialization. This builtin should be used by those that want to use rego objects for signing during policy evaluation. Notice that since these are objects canonicalization *is applied* and all whitespace is removed.
+``io.jwt.encode_sign()`` takes three Rego Objects as parameters and returns their JWS Compact Serialization. This builtin
+ should be used by those that want to use rego objects for signing during policy evaluation. 
+ 
+> Note that with `io.jwt.encode_sign` canonicalization is applied and **all whitespace is removed for headers and body fields**.
```

Comment:
Is this true? Last time I reviewed the code it looked like `.String()` was being called on the header and payload AST values. This does not remove whitespace. For example:

```
package main

import (
	"fmt"

	"github.com/open-policy-agent/opa/ast"
)

func main() {

	term := ast.MustParseTerm(`{"foo":"bar","baz":"qux"}`)
	fmt.Println(term)

}
```

```
$ go run /home/torin/x.go                                                                                                                   
{"foo": "bar", "baz": "qux"}
```

Reply:
Thats a good point.. that wording was taking from some of the other docs. I think the key here is more around "extra" whitespace. Ex:

```golang
package main

import (
	"fmt"

	"github.com/open-policy-agent/opa/ast"
)

func main() {

	term := ast.MustParseTerm(`{
    "foo":"bar",
    "baz":"qux"
}`)
	fmt.Println(term)

}
```
```
{10:14} /t/test ❯ go run main.go
{"foo": "bar", "baz": "qux"}
```

I'll update the docs to try and explain this better.

Reply:
Latest version changes the wording of this to explain better.